mumd/src/client.rs

```diff
@@ -0,0 +1,55 @@
+use crate::command;
+use crate::network::{tcp, udp, ConnectionInfo};
+use crate::state::State;
+
+use futures_util::join;
+use ipc_channel::ipc::IpcSender;
+use mumble_protocol::{Serverbound, control::ControlPacket, crypt::ClientCryptState};
+use mumlib::command::{Command, CommandResponse};
+use std::sync::{Arc, Mutex};
+use tokio::sync::{mpsc, watch};
+
+pub async fn handle(
+    command_receiver: mpsc::UnboundedReceiver<(
+        Command,
+        IpcSender<mumlib::error::Result<Option<CommandResponse>>>,
+    )>,
+) {
+    let (connection_info_sender, connection_info_receiver) =
+        watch::channel::<Option<ConnectionInfo>>(None);
+    let (crypt_state_sender, crypt_state_receiver) =
+        mpsc::channel::<ClientCryptState>(1);
+    let (packet_sender, packet_receiver) =
```

Comment:
I don't know if it's super relevant to this PR, but we probably want to clarify names like these so that they are more descriptive sometime in the future.